src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -487,6 +487,21 @@
         { "command": "resetFontSize", "keys": "ctrl+0" },
         { "command": "resetFontSize", "keys": "ctrl+numpad_0" },
 
+        // Transparency Effects
+        {          
+            "name": { "key": "SetOpacityParentCommandName" },
+            "commands": [
+                { "command": { "action": "adjustOpacity", "opacity": 0, "relative": false } },
+                { "command": { "action": "adjustOpacity", "opacity": 25, "relative": false } },
+                { "command": { "action": "adjustOpacity", "opacity": 50, "relative": false } },
+                { "command": { "action": "adjustOpacity", "opacity": 75, "relative": false } },
+                { "command": { "action": "adjustOpacity", "opacity": 100, "relative": false } }
+            ]
+
+            
+        },   
+        { "command": "toggleAcrylic", "keys": "ctrl+space" },

```

Comment:
actually I'm gonna block over this one

Reply:
@zadjii-msft @kalimal

@zadjii-msft @KalleOlaviNiemitalo
> actually I'm gonna block over this one

I'm sorry not sure what you mean with that phrase to block over something. I understand that it means you need to think deeply about this, is that correct? 

I think it should have a key bind for sure because:

1. I am a Terminal user and want to be able to do this with a keybind. 
2. @moonunit9 would likely agree, given the initial ask, mentioning keystrokes: "Ability to quickly toggle opacity on/off through keystrokes. It would switch between 100% and custom setting in properties."
3. It is an Acrylic Toggle so you want to be able to switch it quickly a keybind allows for this. 
4. Try it yourself and see what you prefer, I am pretty sure you actually want it to be a keybind too. But ctrl+space might not be the best choice, maybe something else would be better? 

Could we maybe ask other terminal users somewhere about their opinion on this? When I made games I'd just ask the players for stuff like this, people's choice.

Reply:
> > actually I'm gonna block over this one
> 
> I'm sorry not sure what you mean with that phrase: "to block over something".

Basically, in code-review parlance, "blocking" a PR is when a reviewer hits that "Request changes" button in response to an issue with the PR. By actually requesting changes, that prevents the PR from merging, even if two other people do sign off on it. 

----

It's totally fine for users to add a keybinding for this action, but I don't think we should bind one _for them_. They're always free to add their own keybinding in the settings. 

If you put it in `defaults.json` like this, then it'll bind <kbd>ctrl+space</kbd> to this action _for everyone_. Binding it here would prevent any CLI application from reading that key unless the user manually unbinds it. And <kbd>ctrl+space</kbd> is already used by different shells for a variety of things. Best we not steal it from them. 

If you change this instead to just

```json
{ "command": "toggleAcrylic" },
```

It'll still show up in the Command Palette, and in the Settings UI, so users can easily add their own keys.

Reply:
Didn't know I could add new actions for things that had no keys bound to them. 
That wasn't intuitive to me. 

I see it now: 
![image](https://github.com/microsoft/terminal/assets/15957528/b8026a79-c9e0-4787-a250-f6ac1dec808e)

Reply:
@zadjii-msft 

Updated it